spec/lib/hyrax/hirmeos/metrics_tracker_spec.rb

```diff
@@ -59,17 +69,44 @@
         children: nil
       }
       work_json = tracker.work_to_hirmeos_json(work)
-      tracker.submit_to_hirmeos(work.id, work_json)
+      tracker.submit_works_to_hirmeos(work.id, work_json)
       expect(a_request(:post, tracker.translation_base_url + "/works").with(body: structure.to_json)).to have_been_made.at_least_once
     end
 
     it 'does not call the register endpoint if a work is already registered' do
       work_json = tracker.work_to_hirmeos_json(work)
-      tracker.submit_to_hirmeos(work.id, work_json)
+      tracker.submit_works_to_hirmeos(work.id, work_json)
       expect(a_request(:post, tracker.translation_base_url + "/works")).not_to have_been_made
```

Comment:
You should be using string interpolation here: "#{tracker.translation_base_url}/works"